ada_2_fhir-r4/fhir/2_fhir_fhir_include.xsl

```diff
@@ -496,7 +520,8 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
                             <xsl:with-param name="in" select="($identifier[not(@root = $mask-ids-var)], $identifier)[1]"/>
                         </xsl:call-template>
                     </identifier>
-                </xsl:when> <!--<xsl:when test="local-name() = $adaElement and .[@value]">
+                </xsl:when>
+                <!--<xsl:when test="local-name() = $adaElement and .[@value]">
```

Comment:
Should we just remove this commented-out code?

Reply:
Agree!